ratis-server/src/main/java/org/apache/ratis/server/impl/PeerConfiguration.java

```diff
@@ -90,11 +90,8 @@ boolean hasMajority(Collection<RaftPeerId> others, RaftPeerId selfId) {
       if (contains(other)) {
         num++;
       }
-      if (num > size() / 2) {
-        return true;
-      }
     }
-    return false;
+    return num > size() / 2;
```

Comment:
why change this ? looks like equivalent with origin code.

Reply:
After this change, it can handle single-server cluster.
Previous one can not handle such case, since its `others` is empty.

Reply:
`if (contains(selfId)) {
      num++;
  }`
This check should handle single server case.

Reply:
For single-server cluster,  `others` will be empty, `for` loop will not be entered, 
```
       if (num > size() / 2) {		
         return true;		
       }	
```
will be un-reachable.

Even `num = 1` will return `false`